src/utils/DockerUtil.js

```diff
@@ -20,20 +23,23 @@ export default {
   activeContainerName: null,
 
   setup (ip, name) {
-    if (!ip || !name) {
+    if (!ip && !name) {
```

Comment:
I only want to throw an error if both the name and the IP are unavailable. When setting up a socket, I don't need a `name`, I can just use `localhost` (although a bogus name could be passed)
When we pass the VBox setup, the`ip` and the `name` are needed to verify the machine and cert location.